Basic-Recruit-Training/Person.cpp

```diff
@@ -0,0 +1,44 @@
+#include "Person.h"
+#include <iostream>
+#include <string>
+Person::Person(){
+    age_=new int(0);
+    name_="";
+}
+Person::Person(const std::string& name, int age){
+  age_=new int(age);
+  name_=name;
+
+}
+Person::~Person(){
+    delete age_;
+}
+int Person::getAge()const{
+ return *age_;
+}
+void Person::printInfo(){
+std::cout << name_<< std::endl;
```

Comment:
For this function, I'd like you to use the helper functions `getName()` and `getAge()` instead of the member variables.